sdk/appconfiguration/perf-tests/app-configuration/test/appConfigBase.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { PerfStressTest, getEnvVar } from "@azure/test-utils-perfstress";
+import { AppConfigurationClient } from "@azure/app-configuration";
+
+export abstract class AppConfigTest<TOptions> extends PerfStressTest<TOptions> {
+  client: AppConfigurationClient;
+
+  constructor() {
+    super();
+    const connectionString = getEnvVar("APPCONFIG_CONNECTION_STRING");
```

Comment:
What's the difference between the 'constructor' and the 'globalSetup' method? Or are they basically the same in this case?

Reply:
Yes, globalMethods do not do anything here.